views/components/HeroFooter/HeroFooter.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { render, screen } from '@testing-library/react'
+import HeroFooter from '.'
+
+describe('Hero Component', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = render(
+      <MemoryRouter>
+        <HeroFooter />
+      </MemoryRouter>
+    )
+  })
+
+  test('renders Hero Footer', () => {
+    const { container } = wrapper
+
+    const Footer = container.firstChild
+
+    expect(Footer).toBeInTheDocument()
```

Comment:
let's use the `screen` import instead of defining a wrapper. Also, do we need this test if we have the others?

Reply:
[1d6bb22](https://github.com/AMP-SCZ/dpdash/pull/468/commits/1d6bb2230345cbb84bf5aac92821f916e7aaad79)